navigation/libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/instruction/turnlane/TurnLaneAdapter.java

```diff
@@ -55,22 +54,24 @@ public void addTurnLanes(IntersectionLanes[] lanes, String maneuverModifier) {
     notifyDataSetChanged();
   }
 
-  private void setLaneImage(TurnLaneViewHolder holder, IntersectionLanes lane) {
+  private void setLaneImage(TurnLaneViewHolder holder, IntersectionLanes lanes) {
     StringBuilder builder = new StringBuilder();
 
     // Indications
-    if (lane.getIndications() != null) {
-      for (String indication : lane.getIndications()) {
+    if (lanes.getIndications() != null) {
```

Comment:
@ericrwolfe you mind double checking this turn lane logic?

Reply:
Can we not show lane guidance if there's only a single lane indication?

Reply:
(i.e. only a `straight` arrow or a `left` arrow)